src/blob_format.h

```diff
@@ -137,6 +137,10 @@ class BlobFileMeta {
   bool is_obsolete() const { return state_ == FileState::kObsolete; }
   uint64_t discardable_size() const { return discardable_size_; }
 
+  bool gc_mark() const { return gc_mark_; }
+  void SetGCMark() { gc_mark_ = true; }
```

Comment:
maybe `set_gc_mark(bool)` ?

Reply:
emmm, ok, because I just copy it from your version :P, I will correct it